antora-playbook-dev.yml

```diff
@@ -0,0 +1,19 @@
+site:
+  title: TinyMCE Documentation (Dev)
+  url: https://www.tiny.cloud/docs
+  start_page: tinymce::index.adoc
+content:
+  sources:
+  - url: ./
+    branches: HEAD
+    start_path: ./
+urls:
+  html_extension_style: indexify
+ui:
+  bundle:
+    url: https://tiny-cloud-docs-antora-themes-staging.s3.amazonaws.com/ui-bundle.zip
+asciidoc:
+  attributes:
+    tinymce_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/6-dev/tinymce.min.js
```

Comment:
Just wanted to check if this should be pointing to 5-dev instead?

Reply:
Thanks for catching that, and for reminding me about checking the demos during the content migration.